.github/workflows/github-issues-monitor.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         export author=${{ github.event.pull_request.user.login }}
-        export slack_id=$(aws s3 cp s3://mysten-employees-dir/employees.json - | jq --arg AUTHOR "${author}" '.[] | if .github_handle == $AUTHOR then .slack_id else empty end')
```

Comment:
@ebmifa I see the original script where I copied this from uses this `-`. Locally, trying with a `json` file this `-` seems to make the program stall. I am not sure I follow the need for `-`...

Reply:
`-` option in `s3` command just streams the contents of the json file, so that we can just use `jq` to parse it

Reply:
Thanks for explaining that!

Reply:
@ebmifa I reverted that change because it was working fine. The issue was with the if condition from my understanding on how these `if` statements + jobs variables work.